config/install/components.settings.yml

```diff
@@ -0,0 +1,3 @@
+namespace_prefix: 0
```

Comment:
From the module:

>          // Prefix the namespace with original name.
>          // Prevents overriding components with duplicate valid namespaces.

Might be worth looking at later.

Reply:
https://stanfordits.atlassian.net/browse/D8CORE-331

Reply:
interesting....
``` if (!empty($config->get('namespace_prefix'))) {
              $namespace = "{$name}_{$namespace}";
            }

            $this->libraries[] = array(
              'type' => $type,
              'name' => $name,
              'namespace' => $namespace,
              'paths' => $paths,
              'error' => FALSE
            );```

Reply:
so basically all that this does is namespace the library with the module/profile/theme name. So if jumpstart_ui & stanford_basic have the same `component-libraries` defined in the info.yml, then the libraries would be prefixed with the extension name. for example is both the module and the theme have `decanter` `component-libraries` as:
```
component-libraries:
  decanter:
    paths:
      - patterns
```

Then the each extension has its own namespaced library as `jumpstart_ui_decanter` and `stanford_basic_decanter`